### DISCUSSION TITLE
Input Remapper settings

### DISCUSSION THREAD

**Discussion by Aqua1ung** [sezanzeb/input-remapper#499] [Q&A] [answered]
I need to re-build my Ubuntu system, and it would help me a great deal if I could simply save the Input Remapper presets, and then import them into the new installation. Is there a file I could save/preserve that stores IR presets/settings?

**Comment by jonasBoss** [comment, marked as answer]
they are all in `~/.config/input-remapper/` you can just backup the whole folder

**Reply by Aqua1ung** [reply]
Thank you! Found them!
